src/main/java/com/unbappd/enterprise/dao/ReviewsDAOStub.java

```diff
@@ -0,0 +1,18 @@
+package com.unbappd.enterprise.dao;
+
+import com.unbappd.enterprise.dto.Review;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class ReviewsDAOStub implements ReviewsDAO {
+
+    public List<Review> reviewsList = new ArrayList<>();
+
+    @Override
+    public void save(Review review) {
+        reviewsList.add(review);
+    }
+}
```

Comment:
To separate the service and DAO, I created a DAO and stub so saving a review to a list can be performed in the DAO instead of service. Right now you are saving to a list in this stub, but eventually any kind of CRUD related actions that you will use with Hibernate will need to be in the DAO.